src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,580 @@
+        <tr>
+            <th>Default</th>
+            <td>
+                ""
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.textToSpeech({
+    utteranceOpts: {
+        text: "Override other texts"
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `lang` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>lang</code> option gives you the ability to specify the language of the text.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td>
+                Default to the language of the HTML document.
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.textToSpeech({
+    utteranceOpts: {
+        lang: "en-US"
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `voiceURI` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>voiceURI</code> option specifies speech synthesis voice and the location of the speech synthesis service that the web application wishes to use.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td>
+                Default to the user agent default speech service. It varies depending on the browser configuration.
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.textToSpeech({
+    utteranceOpts: {
+        voiceURI: {
+            "default": false,
+            "localService": false,
+            "lang": "en-GB",
+            "name": "Google UK English Male",
+            "voiceURI": "Google UK English Male"
+        }
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <th>See also</th>
+            <td>
+            Method <a href="#getvoices">getVoices()</a>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `volume` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>volume</code> option allows you to adjust the volume of the speech. This is a float value between 0 and 1.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td>
+                1
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.textToSpeech({
+    utteranceOpts: {
+        volume: 0.5
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `rate` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>rate</code> option allows you to define the speed at which the text should be spoken. This is a float value between 0 and 10.
```

Comment:
If this is a float value, maybe the example should reinforce that fact by actually using a float value.